src/test/java/seedu/address/model/person/PersonTest.java

```diff
@@ -19,11 +18,11 @@
 
 public class PersonTest {
 
-    @Test
-    public void asObservableList_modifyList_throwsUnsupportedOperationException() {
-        Person person = new PersonBuilder().build();
-        assertThrows(UnsupportedOperationException.class, () -> person.getTags().remove(0));
-    }
+    //    @Test
+    //    public void asObservableList_modifyList_throwsUnsupportedOperationException() {
+    //        Person person = new PersonBuilder().build();
+    //        assertThrows(UnsupportedOperationException.class, () -> person.getTags().remove(0));
+    //    }
```

Comment:
Should we delete commented out code to adhere to the coding standard?

Reply:
I think we should delete as tags field are removed from Person class. If need be we could bring back the code by referencing git later on.